index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Panda</title>
```

Comment:
❗️ 수정요청
`title은 "판다마켓"로 설정합니다.` 요구사항을 구현하기 위해 아래처럼 수정하시는 것을 추천드려요~

```suggestion
    <title>판다마켓</title>
```